src/spikeinterface/curation/auto_merge.py

```diff
@@ -1,53 +1,74 @@
 from __future__ import annotations
 
+from typing import Tuple
 import numpy as np
+import math
 
-from ..core import create_sorting_analyzer
-from ..core.template import Templates
+try:
+    import numba
+
+    HAVE_NUMBA = True
+except ImportError:
+    HAVE_NUMBA = False
+
+from ..core import SortingAnalyzer, Templates
 from ..core.template_tools import get_template_extremum_channel
 from ..postprocessing import compute_correlograms
 from ..qualitymetrics import compute_refrac_period_violations, compute_firing_rates
 
 from .mergeunitssorting import MergeUnitsSorting
+from .curation_tools import resolve_merging_graph
+
+
+_possible_presets = ["default", "xcontamination", "temporal_splits", "feature_neighbors"]
```

Comment:
I would put xcontaminations with an s, but maybe this is too much. And then maybe default is also not really informative, when all the others start to be. What about xcorrelograms?

Reply:
I am agree. We could put "legacy" or better name.